stor/s3.py

```diff
@@ -113,63 +113,19 @@ def _get_s3_transfer(config=None):
 
 
 class S3DownloadLogger(utils.BaseProgressLogger):
-    def __init__(self, total_download_objects):
-        super(S3DownloadLogger, self).__init__(progress_logger)
-        self.total_download_objects = total_download_objects
-        self.downloaded_bytes = 0
-
-    def update_progress(self, result):
-        """Tracks number of bytes downloaded."""
-        self.downloaded_bytes += (os.path.getsize(result['dest'])
-                                  if not utils.has_trailing_slash(result['source']) else 0)
-
-    def get_start_message(self):
-        return 'starting download of %s objects' % self.total_download_objects
-
-    def get_finish_message(self):
-        return 'download complete - %s' % self.get_progress_message()
-
-    def get_progress_message(self):
-        elapsed_time = self.get_elapsed_time()
-        formatted_elapsed_time = self.format_time(elapsed_time)
-        mb = self.downloaded_bytes / (1024 * 1024.0)
-        mb_s = mb / elapsed_time.total_seconds() if elapsed_time else 0.0
-        return (
-            '%s/%s\t'
-            '%s\t'
-            '%0.2f MB\t'
-            '%0.2f MB/s'
-        ) % (self.num_results, self.total_download_objects, formatted_elapsed_time, mb, mb_s)
+    logger = progress_logger
+
+    def add_result(self, result):
+        self.update_progress(num_objects=1, addl_bytes=os.path.getszie(result['dest']))
```

Comment:
Is this a typo?  `getszie`

Reply:
hmmm... maybe I didn't let the s3 version bubble up long enough..